### DISCUSSION TITLE
Auto open new tab when closing last tab

### DISCUSSION THREAD

**Discussion by ahXN00** [Floorp-Projects/Floorp#1198] [Feature Request]
Hi,
If possible can you please add an option like ,when i close last tab it will auto open a new tab.In this way browser will not close when i close last tab.

Screenshot from Brave browser.

![image](https://github.com/Floorp-Projects/Floorp/assets/53382135/94de75f9-bbe9-465c-ab2e-a93d15a8d996)

**Comment by doxxx** [comment]
You can do this with the advanced preference `browser.tabs.closeWindowWithLastTab`. It really just need to be exposed in the Floorp settings.

**Reply by CutterKnife** [reply]
Pull requests are welcome!

**Reply by ahXN00** [reply]
> You can do this with the advanced preference `browser.tabs.closeWindowWithLastTab`. It really just need to be exposed in the Floorp settings.

Thank your for the reply

**Reply by doxxx** [reply]
Pull request: https://github.com/Floorp-Projects/Floorp/pull/1211
